scripts/ssl/EXAMPLE-CERT-CREATION.md

```diff
@@ -4,6 +4,10 @@
 
 Please execute the following script `bash create-tls-certs.sh` to create all necessary certificates for the complete setup of all related components.
 
+## Adoption
```

Comment:
Adoption means start to use something. I guess you need adaptation = adjust something, somehow, to your needs in this case

Reply:
Yes, it's a typo :) I'll fix it.